README.md

```diff
@@ -38,6 +38,10 @@ module "db" {
   
   maintenance_window = "Mon:00:00-Mon:03:00"
   backup_window      = "03:00-06:00"
+
+  # Enhanced Monitoring
+  monitoring_interval = "${var.monitoring_interval}
+  monitoring_role_arn = "${var.monitoring_interval == "0" ? "" : aws_iam_role.rds_monitoring.arn}"
```

Comment:
Similar should be in usage section. Most users want to just copy-paste this from readme file and put as little effort as possible to get it up, but line 44 will make them think that they need to create IAM role somehow.

Keep this section as easy as possible and make it look as end result:
```
monitoring_interval = "10"
monitoring_role_arn = "arn:aws:i-don't-know-but-you-know-what-to-write-here:12345678"
```

No need to use variables here - again, for simplicity.

Reply:
Yeah, I didn't mean to include that.

But, there's no magic role arn you can include - you have to create the role. Not sure how best to document it.

Reply:
I think it will be best to make an example, where you create everything what is needed for it. And link to this example on the main readme page.

Maybe soon we can also start section "Features", where we can highlight what this module can do. Something like:
```
Features:
* Create RDS resources with most of the options supported
* Support [enhanced monitoring](example/...). This module does not create IAM role for you. Check example code for more details.
```

Reply:
OK, I'll pull the params from the README and fix pu the example.